### DISCUSSION TITLE
Outdent items after zooming-in

### DISCUSSION THREAD

**Discussion by vslinko** [vslinko/obsidian-zoom#3] [Ideas]
**Is your feature request related to a problem? Please describe.**
There are unattached vertical lines after zooming-in.

**Describe the solution you'd like**
Outdent items after zooming-in and hide unattached vertical lines.

**Describe alternatives you've considered**
Leave it as it is.

**Comment by rushikb** [comment]
this would be fantastic to have— I'm coming from roam and have a habit of going fairly deep into a thread when working, the vertical lines at the beginning are a bit distracting

**Comment by vslinko** [comment]
Discussion was transferred from https://github.com/vslinko/obsidian-outliner to https://github.com/vslinko/obsidian-zoom

**Comment by maanemann** [comment]
Okay, if i understand the request right, this is what i miss too. When you have a lot of indentation levels, and your screen isn't too wide (eg. a phone screen) you suffer from the lack of horizontal scrolling. In the end Obsidian just breaks down with a weird, faulty layout, as there's no more room. This was actually what i got this plug-in for, to solve this problem. I'm used to Checkvist, where zooming in automatically moves the zoomed parent line all the way to the left edge of the viewport, as if it didn't have any parents (which it doesn't, when it's zoomed). This way you can turn any line/family into a temporary note on its' own. Hope this feature makes it's way to this plug-in!
